recipe/increase_build_number.py

```diff
@@ -1,13 +1,27 @@
 #!/usr/bin/env python
 
 import re
+import requests
 
+
+# read meta.yaml file
 with open("recipe/meta.yaml", "r") as f:
     text = f.read()
 
+# increase build number
 build_number_old = int(re.search(r"(?<={% set build_number = )\d+", text)[0])
 build_number_new = build_number_old + 1
 print(f"set build_number from build_number {build_number_old} to {build_number_new}")
 text = re.sub(r"(?<={% set build_number = )\d+", str(build_number_new), text)
+
+# set sha256sum
+url = "https://github.com/FreeCAD/FreeCAD-Bundle/releases/download/weekly-builds/freecad_source_sha256sum.txt"
```

Comment:
```suggestion
url = "https://github.com/FreeCAD/FreeCAD-Bundle/releases/download/weekly-builds/freecad_source.tar.gz-SHA256.txt"
```

sorry I changed the name to match the rest of the SHA256.txt files we have when I fixed the source package action